src/services/doppler-api-client.ts

```diff
@@ -66,13 +66,12 @@ export class HttpDopplerAPIClient implements DopplerAPIClient {
     apikey: string,
   ): Promise<ResultWithoutExpectedErrors<SubscriberList>> {
     try {
-      // until jwtToken is enabled disable use of variable rule
-      // eslint-disable-next-line
-      const { jwtToken, userAccount } = this.getDopplerAPIConnectionData();
+      const { userAccount } = this.getDopplerApiConnectionData();
+      const jwtToken = apikey ? apikey : this.getDopplerApiConnectionData().jwtToken;
```

Comment:
Que opinás acerca de no aceptar el parámetro apikey y directamentes sacarlo de experimentalFeatures.

Reply:
I thought about this, but then I chickened out

Reply:
Haha, nah, let me know and we could pair programming it if you need some encouragement.